cloud/azure/bin/lib/key_vault.sh

```diff
@@ -16,8 +16,7 @@ function key_vault::create_vault() {
   az keyvault create \
     --name "${3}" \
     --resource-group "${1}" \
-    --location "${2}" \
-    --enable-rbac-authorization
```

Comment:
I don't remember now but I think it was because it was throwing an error and I removed it as a test. But I just added it back and it's working now so maybe it was a transient issue.